src/ui/version.coffee

```diff
@@ -15,7 +15,7 @@ check = (noAlert = false)->
         return console.log err if err
         body = JSON.parse body
         tag = body.tag_name
-        releasedVersion = tag.substr(1) # remove first "v" char
```

Comment:
getting error on this line, replaced by following one

![image](https://cloud.githubusercontent.com/assets/211358/21097501/a323f818-c05c-11e6-9dd4-7d2279bbf855.png)